docker/app/Dockerfile

```diff
@@ -20,13 +23,11 @@ COPY typings ./typings/
 COPY webpack.config.js webpack-dev.config.js webpack-prd.config.js tsconfig.json tslint.json ./
 
 # copy in src local files
+# Note: *.html files in src/angular-app aren't necessary for webpack compilation, however changes to HTML files will invalidate this layer
 COPY src/angular-app ./src/angular-app
-COPY src/appManifest ./src/appManifest
-COPY src/js ./src/js
-COPY src/json ./src/json
 COPY src/sass ./src/sass
-COPY src/service-worker ./src/service-worker
-COPY src/Site/views ./src/Site/views
```

Comment:
Removing the appManifest and service-worker lines will only work after #1247 is merged

Reply:
I merged it, you may rebase now